java-manta-it/src/test/java/com/joyent/manta/client/multipart/EncryptedServerSideMultipartManagerIT.java

```diff
@@ -444,10 +444,10 @@ public void cantUploadSmallMultipartString() throws IOException {
             assert exception.getCause() instanceof IllegalStateException;
```

Comment:
Well that is the `assert` keyword right?  One of the test methods would almost certainly clearer.  I think there might have been some testng vs junit confusion at the time of writing, but either way I don't see why it could not be cleaned up now.